src/exec/exec.go

```diff
@@ -0,0 +1,78 @@
+package exec
+
+import (
+	"context"
+	"math"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"gopkg.in/op/go-logging.v1"
+
+	"github.com/thought-machine/please/src/core"
+	"github.com/thought-machine/please/src/fs"
+	"github.com/thought-machine/please/src/process"
+)
+
+var log = logging.MustGetLogger("exec")
+
+func Exec(state *core.BuildState, label core.BuildLabel, cmdArgs []string, shareNetwork bool) {
```

Comment:
Comment plz? Not sure why the linter is not moaning at you about it...

Reply:
I wondered the same a while ago... will add.